### DISCUSSION TITLE
[Support]: How do you delete a camera in Frigate Plus?

### DISCUSSION THREAD

**Discussion by rwlove** [blakeblackshear/frigate#12342] [General Support] [answered]
### Describe the problem you are having

I renamed some cameras and then uploaded images. Some of the uploaded images had the old camera name and thus created a new, but incorrect/old-name, camera entity in Frigate Plus. How can I delete these incorrect cameras/images?

### Version

0.14

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Other Linux

### Install method

Docker CLI

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
there is no way to do this currently, it is on the roadmap for frigate+
